weldx_widgets/widget_groove_sel.py

```diff
@@ -34,6 +33,107 @@
 ]
 
 
+class WidgetSTLExport(WidgetMyVBox):
```

Comment:
I think we can just call it `WidgetCADExport` :)

Reply:
Right, that name is more general.